executor/executor.cc

```diff
@@ -578,8 +598,14 @@ void reply_execute(int status)
 	reply.magic = kOutMagic;
 	reply.done = true;
 	reply.status = status;
+#if GOOS_windows
+	if (!WriteFile((HANDLE)_get_osfhandle(248), &reply, sizeof(reply), NULL, NULL)) {
```

Comment:
s/248/kOutPipeFd/

Reply:
Okay Will use the variables rather than the value.